inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_request_internal.hpp

```diff
@@ -233,23 +233,23 @@ class InferRequestInternal : virtual public IInferRequestInternal {
     InferenceEngine::InputsDataMap _networkInputs;  //!< Holds information about network inputs info
     InferenceEngine::OutputsDataMap _networkOutputs;  //!< Holds information about network outputs data
     InferenceEngine::BlobMap _inputs;  //!< A map of network input blobs
```

Comment:
let's fix the comment that input and outputs - are user passed blobs.

Reply:
fixed